src/components/forms/validate-identity-form.tsx

```diff
@@ -20,6 +22,7 @@ const ValidateIdentityForm: React.FC<{
   const [validationCode, setValidationCode] = React.useState<string>("");
   const [flashMessage, setFlashMessage] = React.useState<string>("");
   const [formID, setFormID] = React.useState<string>(uuidv4());
+  const [reSentEventId, setReSentEventId] = React.useState<string | null>(null);
```

Comment:
What if we made the page reload (or go on the same path as previously), would the code be simpler?

Reply:
Probably, and it would allow me to add an alert message to :o.